server/src/internalClusterTest/java/org/elasticsearch/snapshots/SnapshotStatusApisIT.java

```diff
@@ -361,6 +361,49 @@ public void testGetSnapshotsMultipleRepos() throws Exception {
         }
     }
 
+    public void testGetSnapshotsWithSnapshotInProgress() throws Exception {
+        createRepository("test-repo", "mock", Settings.builder().put("location", randomRepoPath()).put("block_on_data", true));
+
+        createIndex("test-idx-1");
+        ensureGreen();
+
+        logger.info("--> indexing some data");
+        for (int i = 0; i < 100; i++) {
```

Comment:
I think you can just use `createIndexWithContent("test-idx-1");` here to get an index with some content. Saving you all the steps including `refresh()`

Reply:
OK, I've done that.